LICENSE

```diff
@@ -1,17 +1,17 @@
 BSD 2-Clause License
 
-Copyright (c) 2020, Hypothesis
+Copyright (c) 2022, Hypothesis
```

Comment:
I'm always curious about whether this is legit or not. Lots of people do it, but I have a suspicion you are supposed to leave the older date for it to be as valid as it can be. i.e. updating the date actually removes our copy enforcement for the previous two years. 

Not like we care in particular, but thought it was interesting.

Reply:
We should maybe change this to a `cookiecutter.json` variable that defaults to the current year, the default being generated once when the project is created not regenerated every time `make template` is run. Currently the cookiecutter [uses the current year](https://github.com/hypothesis/cookiecutters/blob/23a0f5558e53098a58b4b068b681b86884d94b76/_shared/project/LICENSE#L3) for this so each year it will want to update the `LICENSE` file in every project